corpora/proverbs.dep

```diff
@@ -27,3 +28,979 @@
+# text[eng] = If you live a lot (of time), you will see a lot.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Көп>"
+	"көп" adv @advmod #1->2
+"<жашасаң>"
+	"жаша," v iv gna_cond p2 sg @advcl #2->5
+"<,>"
+	"," cm @punct #3->5
+"<көптү>"
+	"көп" adj subst acc @obj #4->5
+"<көрөсүң>"
+	"көр" v tv aor p2 sg @root #5->0
+"<.>"
+	"." sent @punct #6->5
+
+# sent_id = proverbs:19
+# author:id = Сыргабековна&Сабыровна:348
+# text = Оюндан от чыгат.
+# text[eng] = From playing comes fire.
+# label_cg = JNW_checked
+# label_dep = revised_1time
+"<Оюндан>"
+	"оюн" n abl @obl #1->3
+"<от>"
+	"от" n nom @nsubj #2->3
+"<чыгат>"
+	"чык" v iv aor p3 sg @root #3->0
+"<.>"
+	"." sent @punct #4->3
+
+# sent_id = proverbs:20
+# author:id = Сыргабековна&Сабыровна:249
+# text = Кайра кирер эшигиңди катуу жаппа.
+# text[eng] = Don't close a door firmly that you might go back through. (="Don't burn bridges.")
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Кайра>"
+	"кайыр" v tv gna_impf @advmod #1->5
+"<кирер>"
+	"кир" v iv gpr_fut @amod #2->3
+"<эшигиңди>"
+	"эшик" n px2sg acc @obj #3->5
+"<катуу>"
+	"катуу" adv @advmod #4->5
+"<жаппа>"
+	"жап" v tv neg imp p2 sg @root #5->0
+"<.>"
+	"." sent @punct #6->5
+
+# sent_id = proverbs:21
+# author:id = Сыргабековна&Сабыровна:283
+# text = Колуң менен кылганды, мойнуң менен тартасың.
+# text[eng] = You pull with your neck what you make with your hands.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Колуң>"
+	"кол" n px2sg nom @obl #1->3
+"<менен>"
+	"менен" post @cc #2->1
+"<кылганды>"
+	"кыл" v tv gpr_past subst acc @conj #3->7
+"<,>"
+	"," cm @punct #4->7
+"<мойнуң>"
+	"моюн" n px2sg nom @obl #5->7
+"<менен>"
+	"менен" post @cc #6->5
+"<тартасың>"
+	"тарт" v tv aor p2 sg @root #7->0
+"<.>"
+	"." sent @punct #8->7
+
+# sent_id = proverbs:22
+# author:id = Сыргабековна&Сабыровна:465
+# text = Шашкан шайтандын иши.
+# text[eng] = Hurrying is what a shaytan does. / Hurrying is the devil's work.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Шашкан>"
+	"шаш" v iv ger_past nom @nsubj #1->3
+"<шайтандын>"
+	"шайтан" n gen @nmod:poss #2->3
+"<иши>"
+	"иш" n px3sp nom @root #3->0
+		"э" cop aor p3 sg @cop #4->3
+"<.>"
+	"." sent @punct #5->3
+
+# sent_id = proverbs:23
+# author:id = Сыргабековна&Сабыровна:313
+# text = Куру сөз курсак тойгузбайт.
+# text[eng] = A dry word does not fill a belly.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Куру>"
+	"куру" adj @amod #1->2
+"<сөз>"
+	"сөз" n nom @nsubj #2->4
+"<курсак>"
+	"курсак" n nom @obj #3->3
+"<тойгузбайт>"
+	"той" v iv caus neg aor p3 sg @root #4->0
+"<.>"
+	"." sent @punct #5->4
+
+# sent_id = proverbs:24
+# author:id = Сыргабековна&Сабыровна:333
+# text = Мурунку көч кайда барса, кийинки көч да ошол жакка барат.
+# text[eng] = Where the former migration goes, the next migration will go there too.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Мурунку>"
+	"мурун" adv attr @amod #1->2
+"<көч>"
+	"көч" n nom @csubj #2->4
```

Comment:
I think just `@nsubj`.  `@csubj` is for when the dependent is a (verbal) clause; here it's just a noun phrase.